### PR TITLE
moved toolbar padding to upper level

### PR DESCRIPTION
**Pull request by edewit**
re opened because of faulty rebase

### CODE REVIEW

**Comment by edewit** [comment]
@SMahil I only moved the exiting padding from work item to upper level if there needs to be more padding in some cases this would have been an other issue IMO

**Comment by joshuawilson** [comment]
Thanks @edewit 
[test]

File: src/app/app.component.scss [hunk 1]
```diff
@@ -14,3 +14,12 @@
 	border-top: 1px solid $color-pf-blue-300;
 	margin-top: 2%;
 }
+
+.app {
+	padding: em(96) 0 em(66) 0;
```
Comment (SMahil) [inline src/app/app.component.scss]: when the user is no logged in, padding-bottom should be 0. Currently we are using  .containerPadd-nologin{ padding-bottom: 0; }

As we have moved the padding to outer element , we need padding-bottom:0 in that scenario for same element

**Comment by joshuawilson** [comment]
@SMahil can you take a look at this please. It appears that the position of contents of Dashboard and Learn has been pushed down. This is under the Home tab.

**Comment by joshuawilson** [comment]
@edewit @SMahil I would rather move forward on this, so I merged it. Please submit new issues and PRs for things that need fixing.

**Pull request merged**